app/src/main/java/com/github/multimatum_team/multimatum/LocalDateTimeSerializer.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.multimatum_team.multimatum
```

Comment:
Same for this class, I think it would make sense to put both the serializer and deserialized in the same file too.